src/Configurator.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of the KongAdminApi package.
+ *
+ * (c) Unikorp <https://github.com/unikorp>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Unikorp\KongAdminApi;
+
+use Unikorp\KongAdminApi\Node;
+
+/**
+ * configurator
+ *
+ * @author VEBER Arnaud <https://github.com/VEBERArnaud>
+ */
+class Configurator implements ConfiguratorInterface
+{
+    /**
+     * base uri
```

Comment:
sure, was to lazy to describe the function comportement.
Gonna change them to something more useful